sdk/python/setup.cfg

```diff
@@ -16,5 +17,7 @@ select = B,C,E,F,W,T4
 exclude = .git,__pycache__,docs/conf.py,dist,feast/protos
 
 [mypy]
-files=feast,test
+files=feast,tests
 ignore_missing_imports=true
+# ignore mypy error from generated protos modules
+exclude=feast/protos/*
```

Comment:
Why is this suddenly necessary?

Reply:
@woop the generated protos directories are not recognized as valid python packages because they are missing the `__init__.py` files. As such, lines such as 
```python
from feast.protos.feast.serving import Connector_pb2 as feast_dot_serving_dot_Connector__pb2
```
are not valid since `Connector_pb2` (module) cannot be imported from `serving` (package). A better solution would be to add empty `__init__.py` files in this generated protos directory tree. 

I didn't spend time to take a look at it properly to see where this could be done or if there is a cleaner way to solve it.

WDYT?

Reply:
mypy error:
```
feast/protos/feast/serving/Connector_pb2_grpc.py:5: error: Module "feast.protos.feast.serving" has no attribute "Connector_pb2"
```

Reply:
We used to have `__init__.py` files there but for some reason it was removed after 0.19 was released.

Reply:
@tpvasconcelos I've created https://github.com/feast-dev/feast/pull/2410/files to generate these `__init__` files; you may need to rebase after that's merged in.

Reply:
@achals just pinging you on this ☝️

Reply:
@tpvasconcelos will follow up for you.

Reply:
@tpvasconcelos ugh sorry. I totally missed this. Thanks for flagging @woop as well. I'll get this fixed.

Reply:
Actually this shouldn't be needed, because `make install-python-ci-dependencies` does trigger the generation of python protos as expected. So I'm not sure where the error is coming from. I'm going to checkout your PR locally and see if I can reproduce the error.

Reply:
@tpvasconcelos I was able to reproduce this issue locally, looks like it was a combination of a missing `__init__.py` file when running py 3.7. I'm going to send out a PR to fix it.

Reply:
#2433